backend/libraries/types/can.did

```diff
@@ -1521,6 +1521,11 @@ type AccessGate = variant {
     SnsNeuron : SnsNeuronGate;
     Payment : PaymentGate;
     TokenBalance : TokenBalanceGate;
+    Composite : record {
```

Comment:
I thought we decided to opt for a list of gates with a bool to say whether it was any or all in the end?